engine/crates/composition/tests/composition/inaccessible_basic/federated.graphql

```diff
@@ -56,9 +56,9 @@ type Series {
 
 type New {
     name: String! @inaccessible
```

Comment:
The changes in the existing tests are there to avoid these schemas to become invalid with the new rule.